target/linux/ramips/dts/mt7621_beeline_smartbox-flash.dts

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "beeline,smartbox-flash", "mediatek,mt7621-soc";
+	model = "Beeline SmartBox Flash";
+
+	aliases {
+		label-mac-device = &gmac0;
```

Comment:
I think this can be dropped.

Reply:
You're absolutely right. Dropped and tested. All is ok.